test_files/jsdoc_types/jsdoc_types.ts

```diff
@@ -6,7 +6,7 @@
 import * as module1 from './module1';
 import {ClassOne, value, ClassOne as RenamedClassOne, ClassTwo as RenamedClassTwo, Interface, ClassWithParams} from './module2';
 import DefaultClass from './default';
-import {NeverTyped} from './nevertyped';
+import {NeverTyped, NeverTypedTemplated} from './nevertyped';
```

Comment:
This adds some test coverage, but is a red herring - the problem turned out to be the type-vs-value dichotomy in the test below.